lib/modules/platform/gitlab/index.ts

```diff
@@ -284,7 +288,8 @@ export async function initRepo({
       );
       throw new Error(REPOSITORY_ARCHIVED);
     }
-    if (res.body.mirror) {
+
+    if (res.body.mirror && includeMirrors !== true) {
```

Comment:
@viceice What if we remove this if altogether?

1. If it's been autodiscovered then we know the includeMirrors flag must be true already
2. If the user has run it directly, they clearly want to include it

Reply:
Let's do it later